src/main/java/software/amazon/msk/auth/iam/internals/MSKCredentialProvider.java

```diff
@@ -75,6 +75,7 @@
  * <a href="https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html">Working with AWS Credentials</a>
  */
 public class MSKCredentialProvider implements AWSCredentialsProvider, AutoCloseable {
+    public static final String AWS_REGION = "awsRegion";
```

Comment:
Can we move this below the logger line and along with all the other string constants ?